### ISSUE TITLE
2embed series support discontinuation

### ISSUE THREAD

**Issue by toobeeh** [toobeeh/Roam#1]
seems like 2embed supports no longer sieries.

alternative that might be worth checking out:
https://embed.smashystream.com/

not sure about accessibility for firetv on this one

**Comment by toobeeh** [comment]
the only other alternative would be https://www.superembed.stream/movie-streaming-api.html
but theyve discontinued their json api, and have a captcha in their default player

**Comment by toobeeh** [comment]
smashystream uses jsplayer library for embedding. 
it would suit if:
- the play button in the landing screen is hit automatically
- the palyer config contains "native controls" as true (in the frame of the embed url)
![image](https://github.com/toobeeh/Roam/assets/66030584/bb77c2fe-c41c-460e-a29f-2eb85f3e089a)

**Comment by toobeeh** [comment]
another solid api: (used on bflix.to)
https://vidsrc.to/#top

would have to test accessibility, but it should have auto-play and subtitles should be able to pre-set via localStorage in the app
